### PR TITLE
Documentation subpackage

### PR DESCRIPTION
**Pull request by jparisu**
_Bracnh over #17_ 

Add Documentation subpackage with empty documentation

### CODE REVIEW

File: amlip_docs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```
Comment (juanlofer-eprosima) [inline amlip_docs/CMakeLists.txt:1]:
```suggestion
# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

File: amlip_docs/cmake/packaging/Config.cmake.in [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+set_and_check(@PROJECT_NAME@_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
+set_and_check(@PROJECT_NAME@_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
+
+if(NOT fastcdr_FOUND)
+    find_package(fastcdr)
+endif()
+
+if(NOT fastrtps_FOUND)
+    find_package(fastrtps)
+endif()
```
Comment (juanlofer-eprosima) [inline amlip_docs/cmake/packaging/Config.cmake.in:17]: Is this actually required?

File: amlip_docs/cmake/test/cmake_test_utils.cmake [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+            "${PROJECT_SOURCE_DIR}/src/cpp/**/*.hxx"
+            "${PROJECT_SOURCE_DIR}/test/TestUtils/**/*.cpp"
+            "${PROJECT_SOURCE_DIR}/test/TestUtils/*.cpp"
+        )
+    set(NEW_TEST_SOURCES "${TEST_SOURCES};${LIBRARY_SOURCES}")
+    set(TEST_SOURCES ${NEW_TEST_SOURCES} PARENT_SCOPE)
+
+endfunction()
+
+# Return a list of all hpp files required by ddsrouter library
```
Comment (juanlofer-eprosima) [inline amlip_docs/cmake/test/cmake_test_utils.cmake:99]: change

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: amlip_docs/cmake/packaging/Config.cmake.in (hunk 2 above)
Comment (jparisu) [reply amlip_docs/cmake/packaging/Config.cmake.in:17]: This file has been removed

File: amlip_docs/cmake/test/cmake_test_utils.cmake [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+    else(WIN32)
+        # dummy
+        function(get_win32_path_dependencies)
+        endfunction(get_win32_path_dependencies)
+    endif(WIN32)
+
+endmacro()
+
+# Return a list of all cpp and hpp files required by ddsrouter library
```
[the remark this answers is not in the document]
Comment (jparisu) [reply amlip_docs/cmake/test/cmake_test_utils.cmake:79]: _This file has been removed_

File: amlip_docs/cmake/test/cmake_test_utils.cmake (hunk 3 above)
Comment (jparisu) [reply amlip_docs/cmake/test/cmake_test_utils.cmake:99]: _This file has been removed_

**Comment by jparisu** [comment]
This PR requires the PR https://github.com/eProsima/DDS-Router/pull/180 to be merged in DDS Router main.

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**